bundles/framework/layerselector2/service/layerlist.js

```diff
@@ -3,6 +3,34 @@
         function () {
             this.layerlistFilterButtons = {};
             this.currentFilter = null;
+            const me = this;
+            this.mutator = {
```

Comment:
Move mutator creation to it's own function, away from the constructor function.

Reply:
Otherwise looks good for merging

Reply:
Changed as suggested.